packages/common-db/src/lib/common/tools.ts

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+// we currently have no typings so ignore
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import {tools} from '@iobroker/js-controller-common';
+
+/**
+ * Allows to find out if a given states dbType offers a server or not
+ * @param dbType database type
+ * @returns true if a server class is available
+ */
+export function statesDbHasServer(dbType: string): boolean {
+    try {
+        // eslint-disable-next-line @typescript-eslint/no-var-requires
+        return !!require(`@iobroker/db-states-${dbType}`).Server;
+    } catch {
+        throw new Error(`Installation error or unknown states database type: ${dbType}`);
+    }
+}
+
+/**
+ * Allows to find out if a given objects dbType offers a server which runs on this host and listens (locally or globally/by IP)
+ * @param dbType database type
+ * @param host configured db host
+ * @param checkIfLocalOnly if true the method checks if the server listens to local connections only; else also external connection options are checked
+ * @returns true if a server listens on this host (locally or globally/by IP)
+ */
+export function isLocalObjectsDbServer(dbType: string, host:string, checkIfLocalOnly=false): boolean {
```

Comment:
My personal preference is specifying types for parameters even if they have a default value: 
```
checkIfLocalOnly: boolean = false
```
This way you can't forget when changing the default. Might require disabling an ESLint rule though.